commands/builds/create.js

```diff
@@ -83,7 +90,8 @@ module.exports = {
   flags: [
     { name: 'source-url', description: 'Source URL that points to the tarball of your application\'s source code', hasValue: true },
     { name: 'tar', description: 'Path to the executable GNU tar', hasValue: true },
-    { name: 'version', description: 'Description of your new build', hasValue: true }
+    { name: 'version', description: 'Description of your new build', hasValue: true },
+    { name: 'exclude-vcs-ignore', description: 'Exclude files ignores by VCS (.gitignore, ...) from the build', hasValue: true }
```

Comment:
I see it [here](https://devcenter.heroku.com/articles/cli-style-guide#description).
Fixed in c8e7bdf22f8a8ebfd7fc8eaf75a5a9552522c1b4.